.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## PR summary
+<!-- please include a brief summary of the changes in this PR -->
+
+
+## PR Checklist
+
+### General checklist
+Please make sure that your PR fulfills the following requirements:  
+- [ ] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+- [ ] Dependencies have been updated as needed
+- [ ] .secrets.baseline updated as needed?
+
+#### Checklist for adding a new validation rule:
+- [ ] Added new validation rule definition (packages/ruleset/src/rules/*.js, index.js)
+- [ ] Added new validation rule implementation (packages/ruleset/src/functions/*.js, updated index.js)
+- [ ] Added new rule to default configuration (packages/ruleset/src/ibm-oas.js)
+- [ ] Added tests for new rule (packages/ruleset/test/*.test.js)
+- [ ] Added docs for new rule
+
+#### Checklist for removing an old validation rule:
+- [ ] Removed old rule implementation (packages/validator/src/plugins/validation/*.js)
+- [ ] Removed adjusted testcases (packages/validator/test)
+- [ ] Updated default configuration (packages/validator/src/.defaultsForValidator.js)
```

Comment:
Should we explicitly note the "deprecated" field in this file? Might be easy to miss